packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -248,6 +249,9 @@ class BottomNavigationBar extends StatefulWidget {
   /// backgroundColor will splash and overwrite this color.
   final Color backgroundColor;
 
+  /// The color of the active [BottomNavigationBar] item.
+  final Color activeBackgroundColor;
```

Comment:
Nit: The docs here should specify the default behavior.

Reply:
added it to the description

Reply:
And be in sentence form. :)
```
/// Defaults to transparent when null.
```